config/config.go

```diff
@@ -2,26 +2,29 @@
 package config
 
 import (
-	"sync"
-
+	"errors"
 	"github.com/companieshouse/gofigure"
+	"log"
+	"sync"
 )
 
 var cfg *Config
 var mtx sync.Mutex
 
 // Config defines the configuration options for this service.
 type Config struct {
-	BindAddr          string   `env:"BIND_ADDR" flag:"bind-addr" flagDesc:"Bind address"`
-	BrokerAddr        []string `env:"KAFKA_BROKER_ADDR" flag:"broker-addr" flagDesc:"Kafka broker address"`
-	SchemaRegistryURL string   `env:"SCHEMA_REGISTRY_URL" flag:"schema-registry-url" flagDesc:"URL for Schema Registry"`
-	OfficerDeltaTopic string `env:"OFFICER_DELTA_TOPIC" flag:"officer-delta-topic" flagDesc:"Topic for the officer delta"`
+	BindAddr          string    `env:"BIND_ADDR" flag:"bind-addr" flagDesc:"Bind address"`
+	BrokerAddr        []string  `env:"KAFKA_BROKER_ADDR" flag:"broker-addr" flagDesc:"Kafka broker address"`
+	SchemaRegistryURL string    `env:"SCHEMA_REGISTRY_URL" flag:"schema-registry-url" flagDesc:"URL for Schema Registry"`
+	OfficerDeltaTopic string 	`env:"OFFICER_DELTA_TOPIC" flag:"officer-delta-topic" flagDesc:"Topic for the officer delta"`
+	LogLevel		  string  	`env:"LOGLEVEL" flag:"log-level" flagDesc:"Logging level to set i.e error, info, debug"`
```

Comment:
Why do we need log level ? Is it being used anywhere here ?

Reply:
Removed LogLevel as it is not required and is handled by chs.go